bedrock/settings/base.py

```diff
@@ -221,7 +221,6 @@ def data_path(*args):
     "zu",
 )
 
-LOCALES_PATH = DATA_PATH / "locale"
```

Comment:
The next 3 lines can go as well I think

Reply:
Along with the `DOTLANG_FILES` constant below

Reply:
Done!